node/service/src/chain_spec/moonbase.rs

```diff
@@ -214,7 +214,7 @@ pub fn testnet_genesis(
 				.collect(),
 		},
 		pallet_ethereum: EthereumConfig {},
-		pallet_democracy: DemocracyConfig {},
```

Comment:
What does this change do?

Reply:
Just reflecting this change in pallet democracy

![image](https://user-images.githubusercontent.com/18615930/121362260-45b77500-c936-11eb-8dfb-c4d26a1f47f9.png)

Reply:
Just reflecting this change
![image](https://user-images.githubusercontent.com/18615930/121362360-55cf5480-c936-11eb-9285-b8f95beac6f7.png)

Reply:
https://github.com/paritytech/substrate/commit/94679ebd37f4989f8f7803d4f9aa403b7ba9aead